### DISCUSSION TITLE
Comprehensive list of changes/additions

### DISCUSSION THREAD

**Discussion by Yiyio** [Keriew/augustus#464] [Ideas]
Hello,

I was wondering if there is a list of changes with respect to vanilla or julius, or if not, if it could be added somewhere in the manual or readme or wiki.

The pdf manual says "We also removed a number of options from previous versions and have instead made them default Augustus mechanics from Version 2.0.0 onward" 

For example I have seen in older versions there were options to block unconnected granary roads, is this one standard now? Or the 100 year ghosts option also doesn't show (I assume this was made 'permanent').

**Comment by Keriew** [comment]
Manual lists the changes made in the latest version, and the [release page](https://github.com/Keriew/augustus/releases) does so for the previous releases. Though it's not documented from before 1.4.1. I agree that that it could use a more comprehensive changelist somewhere, and I'll work on adding it when I can find time.

**Comment by crudelios** [comment]
We are considering releasing a consolidated manual with all the Augustus changes, that will hopefully fix this issue.
